### PR TITLE
Ce deploy

### PR DESCRIPTION
**Pull request by russojrv-zz**
**What has changed?**

- A new endpoint to impersonate a user has been provided.

### CODE REVIEW

File: services/iam/src/dao/accounts.js [hunk 1]
```diff
@@ -46,7 +46,7 @@ const AccountDAO = {
         });
         EventBusManager.getEventBus().publish(event);
 
-        return account.toJSON();
+        return savedAccount;
```
Comment (weberjm) [inline services/iam/src/dao/accounts.js:49]: When you return the results of account.save(), is this a JSON, a JavaScript object,or a BSON object? Is it 1:1 with the former return value?

**Review by weberjm — COMMENTED** [review]
If you have already confirmed no problems with the return value, this can be merged.

File: services/iam/src/dao/accounts.js (hunk 1 above)
Comment (russojrv-zz) [reply services/iam/src/dao/accounts.js:49]: When we return the result of .save() method, it returns a mongoose document with additional properties/methods, for example, isNew, get() that are used internally in the req.logIn(... I did not want to update this file but I had not other option. I can do a query after I call to the create method but I think it is not necessary just returning this document.

Comment (weberjm) [reply services/iam/src/dao/accounts.js:49]: OK. No worries! Just was curious why it was changed.

**Pull request merged**